client/account-settings/view.js

```diff
@@ -37,9 +38,9 @@ const AccountSettingsRootView = ( { formData, formMeta, actions, noticeActions,
 	const renderContent = () => {
 		if ( ! formData && ! formMeta.isFetching ) {
 			return (
-				<p className="error-message">
+				<ErrorMessage>
```

Comment:
Maybe it's a bit overkill now, but there's no harm in having a component that's only been used twice. It'll probably have more uses in the future though.